nl_server/tests/test_data/climatechange_drought.json

```diff
@@ -1,83 +1,75 @@
 {
-    "SV": [
-      "dc/topic/ClimateChange",
-      "dc/topic/ProjectedClimateExtremes",
-      "Annual_Emissions_GreenhouseGas",
-      "dc/topic/GreenhouseGasEmissionsBySource",
-      "Annual_Generation_Electricity",
-      "Count_DroughtEvent",
-      "Count_HeatEvent",
-      "Annual_Consumption_Electricity",
-      "Count_HeavyRainEvent",
-      "dc/topic/CarbonDioxideEmissionsBySource",
-      "dc/topic/AgricultureEmissionsByGas",
-      "Annual_ExpectedLoss_NaturalHazardImpact"
-    ],
-    "CosineScore": [
-      0.8262420892715454,
-      0.5475385189056396,
-      0.48141297698020935,
-      0.41852420568466187,
-      0.37782663106918335,
-      0.37141096591949463,
-      0.350502610206604,
-      0.34617504477500916,
-      0.3422034978866577,
-      0.33296579122543335,
-      0.33168768882751465,
-      0.3311363756656647
-    ],
-    "SV_to_Sentences": {},
-    "MultiSV": {
-      "Candidates": [
-        {
-          "Parts": [
-            {
-              "QueryPart": "show impact climate change",
-              "SV": [
-                "dc/topic/ClimateChange"
-              ],
-              "CosineScore": [
-                0.7973
-              ]
-            },
-            {
-              "QueryPart": "drought",
-              "SV": [
-                "Count_DroughtEvent"
-              ],
-              "CosineScore": [
-                0.7639
-              ]
-            }
-          ],
-          "AggCosineScore": 0.7806,
-          "DelimBased": false
-        },
-        {
-          "Parts": [
-            {
-              "QueryPart": "show impact climate",
-              "SV": [
-                "dc/topic/ClimateChange"
-              ],
-              "CosineScore": [
-                0.6864
-              ]
-            },
-            {
-              "QueryPart": "change drought",
-              "SV": [
-                "Count_DroughtEvent"
-              ],
-              "CosineScore": [
-                0.7281
-              ]
-            }
-          ],
-          "AggCosineScore": 0.7073,
-          "DelimBased": false
-        }
-      ]
-    }
-  }
\ No newline at end of file
+  "SV": [
+    "dc/topic/ClimateChange",
+    "dc/topic/SDG_13",
+    "dc/topic/ProjectedClimateExtremes",
+    "dc/topic/Temperature",
+    "Annual_Emissions_GreenhouseGas",
+    "dc/topic/GreenhouseGasEmissionsBySource",
+    "Annual_Emissions_GreenhouseGas_NonBiogenic"
```

Comment:
Why do we have fewer SVs?

Reply:
I investigated this. When setting k=40 or something else that's larger than the existing 20 matches in the index, the golden files produced do have more SVs. So this reduction is happened because these golden files were generated using the previous SV match number (20) and since now we have more SV descriptions in the embeddings, fewer SVs are returned for the same sentence matching limit (of 20).

The reason I did not re-update the files with k=40 is that these golden files are only being used for the Multi-SV search tests and they do not look at the single SVs matched for these tests.

Reply:
top-level ones seem to have improved (all topics now)